apps/launcher_app.lua

```diff
@@ -40,7 +40,7 @@ laptop.register_app("launcher", {
 			local x = math.floor((i-1) / c_row_count)*2 + 1
 			local y = ((i-1) % c_row_count)*2 + 1
 			out = out .. mtos.theme:get_image_button(x..','..y..';1,1', 'app', e.name, (e.def.app_icon or 'logo.png'), "", (e.def.app_info or e.name))..
-						mtos.theme:get_label((x-0.3)..','..(y+1), e.def.app_name, "app")
+			mtos.theme:get_image_button((x-.5)..','..(y+1.08)..';2,.4', 'app', e.name, mtos.theme.desktop_icon_label_button	, e.def.app_name)
```

Comment:
get_image_button concatenates 2x textures. The fist one is based on prefix. In this case "app_button+desktop_icon_label_button. This is not necessary.
To use just 1x image please use 

`mtos.theme:get_button((x-.5)..','..(y+1.08)..';2,.4', 'desktop_icon_label', e.name, e.def.app_name)`

Note: the prefix will not be "app" anymore that means the text color fails to fallback. So the "app_textcolor" needs to be renamed to "desktop_icon_label_textcolor" in all themes

Reply:
Please post code you want me to replace with this, because I thought you wanted me to replace most of line 42 and all of 43. That results in no icon, only label. Replacing only 43 gives expected results.

Reply:
42 (the icon) is ok
43: The label should be `mtos.theme:get_button((x-.5)..','..(y+1.08)..';2,.4', 'desktop_icon_label', e.name, e.def.app_name)`
The condition is the prefix is set up in theme:
desktop_icon_label_button - background texture
desktop_icon_label_textcolor - Text color